driver/include/esp_camera.h

```diff
@@ -16,6 +16,7 @@
  *
     static camera_config_t camera_example_config = {
         .pin_reset      = PIN_RESET,
+        .pin_pwdn       = PIN_PWDN,
```

Comment:
please move the pin above reset (it's more inline with logic and example given in esp-who)